### PR TITLE
Hcal relval update 10_0_X digi occupancy error

### PR DESCRIPTION
**Pull request by bcmcms**
Fixed error bars on digi occupancy plots, and added a switch (default off) for v0, v1 TP plots.

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by abdoulline** [comment]
Traceback from TTbar 5.1 (Run 2, 2016) wf :

#5  0x00007fb1f3728234 in MonitorElement::Fill(double, double) () from /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_10_0_X_2017-11-15-1100/lib/slc6_amd64_gcc630/libDQMServicesCore.so

#6  0x00007fb1b1634942 in HcalDigisValidation::analyze(edm::Event const&, edm::EventSetup const&) () from /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_10_0_X_2017-11-15-1100/lib/slc6_amd64_gcc630/pluginValidationHcalDigis.so

@bcmcms  
Brooks, before submitting the changes, it's worth running  
runTheMatrix.py -s &

to make sure nothing is broken.
Now it looks like some MonitorElement is not (well) defined...

**Comment by bcmcms** [comment]
Thanks the advice, @abdoulline. I'll jump in these errors and check out how I defined the new MonitorElement for issues. That matrix test will be very handy for debugging.

**Comment by abdoulline** [comment]
@dmitrijus, @jfernan2, @vanbesien  can someone please type 
"please test"

**Comment by jfernan2** [comment]
please test

**Comment by dmitrijus** [comment]
+1

**Comment by bcmcms** [comment]
Thanks for pointing that out. This should fix of indentation in those files.

**Comment by dmitrijus** [comment]
+1

File: Validation/HcalDigis/src/HcalDigisClient.cc [hunk 1]
```diff
@@ -134,7 +135,11 @@ int HcalDigisClient::HcalDigisEndjob(const std::vector<MonitorElement*> &hcalMEs
     }
 
     std::vector<float> sumphi;
-    for(int depth = 1; depth <= depths; depth++) sumphi.push_back(0.);
+    std::vector<float> sumphie;
```
Comment (davidlange6) [inline Validation/HcalDigis/src/HcalDigisClient.cc]: hi @bcmcms - these are better written as 

std::vector<float> sumphie(depths,0)

File: Validation/HcalDigis/src/HcalDigisClient.cc [hunk 2]
```diff
@@ -174,35 +179,47 @@ int HcalDigisClient::HcalDigisEndjob(const std::vector<MonitorElement*> &hcalMEs
         }
     
         //zero the sumphi vector at the start of each ieta ring
-	for(int depth = 1; depth <= depths; depth++) sumphi[depth-1] = 0;
+    for(int depth = 1; depth <= depths; depth++) {
```
Comment (davidlange6) [inline Validation/HcalDigis/src/HcalDigisClient.cc]: can be sumphi.assign(depths,0)

Comment (davidlange6) [reply Validation/HcalDigis/src/HcalDigisClient.cc]: also the indentation of this loop seems to be incorrect (relative to the lines above). Please make it consistent

Comment (bcmcms) [reply Validation/HcalDigis/src/HcalDigisClient.cc]: Thanks @davidlange6 - they should be looking better now.

**Comment by hatakeyamak** [comment]
It looks to me @bcmcms made updates as David suggested. Hopefully this is good now.

**Comment by davidlange6** [comment]
please test

**Comment by hatakeyamak** [comment]
It looks like it's a DAS error and has nothing to do with the PR itself.

**Comment by hatakeyamak** [comment]
Is there anything @brooks has to do to get this merged?

**Comment by fabiocos** [comment]
please test

**Comment by dmitrijus** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**